src/helpers/xml-builder.js

```diff
@@ -1,2390 +1,2473 @@
+
+            paragraphFragment.import(runOrHyperlinkFragment);
+          }
+        } else {
+          paragraphFragment.import(runOrHyperlinkFragments);
+        }
+      }
+    }
+  } else {
+    // In case paragraphs has to be rendered where vText is present. Eg. table-cell
+    // Or in case the vNode is something like img
+    if (isVNode(vNode) && vNode.tagName === 'img') {
+      const imageSource = vNode.properties.src;
+      let base64String = imageSource;
+      if (isValidUrl(imageSource)) {
+        base64String = await imageToBase64(imageSource).catch((error) => {
+          // eslint-disable-next-line no-console
+          console.warning(`skipping image download and conversion due to ${error}`);
+        });
+
+        if (base64String && mimeTypes.lookup(imageSource)) {
+          vNode.properties.src = `data:${mimeTypes.lookup(imageSource)};base64, ${base64String}`;
+        } else {
+          paragraphFragment.up();
+
+          return paragraphFragment;
+        }
+      } else {
+        // eslint-disable-next-line no-useless-escape, prefer-destructuring
+        base64String = base64String.match(/^data:([A-Za-z-+\/]+);base64,(.+)$/)[2];
+      }
+
+      const imageBuffer = Buffer.from(decodeURIComponent(base64String), 'base64');
+      const imageProperties = sizeOf(imageBuffer);
+
+      modifiedAttributes.maximumWidth =
+        modifiedAttributes.maximumWidth || docxDocumentInstance.availableDocumentSpace;
+      modifiedAttributes.originalWidth = imageProperties.width;
+      modifiedAttributes.originalHeight = imageProperties.height;
+
+      computeImageDimensions(vNode, modifiedAttributes);
+    }
+    const runFragments = await buildRunOrRuns(vNode, modifiedAttributes, docxDocumentInstance);
+    if (Array.isArray(runFragments)) {
+      for (let index = 0; index < runFragments.length; index++) {
+        const runFragment = runFragments[index];
+
+        paragraphFragment.import(runFragment);
+      }
+    } else {
+      paragraphFragment.import(runFragments);
+    }
+  }
+  paragraphFragment.up();
+
+  return paragraphFragment;
+};
+
+const buildGridSpanFragment = (spanValue) =>
+  fragment({ namespaceAlias: { w: namespaces.w } })
+    .ele('@w', 'gridSpan')
+    .att('@w', 'val', spanValue)
+    .up();
+
+const buildTableCellSpacing = (cellSpacing = 0) =>
+  fragment({ namespaceAlias: { w: namespaces.w } })
+    .ele('@w', 'tblCellSpacing')
+    .att('@w', 'w', cellSpacing)
+    .att('@w', 'type', 'dxa')
+    .up();
+
+const buildTableCellBorders = (tableCellBorder) => {
+  const tableCellBordersFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele(
+    '@w',
+    'tcBorders'
+  );
+
+  const { colors, strokes, ...borders } = tableCellBorder;
+  Object.keys(borders).forEach((border) => {
+    if (tableCellBorder[border]) {
+      const borderFragment = buildBorder(
+        border,
+        tableCellBorder[border],
```

Comment:
Can we make the defaults a constant referenced from the defaults? I see it in multiple spots. Also add this change to the options and readme

Reply:
for example `0`, `000000`, `nil`